met-web/tests/unit/components/engagementForm.test.tsx

```diff
@@ -397,7 +397,7 @@ describe('Engagement form page tests', () => {
 
         await waitFor(() => {
             expect(screen.getByText('Create New Contact')).toBeVisible();
-            expect(screen.getByText(mockContact.email)).toBeVisible();
+            expect(screen.getByText('jace@gmail.com')).toBeVisible();
```

Comment:
What made us do this change here? isn't mockContact.email already 'jace@gmail.com'

Reply:
reversed

Reply:
i thought it wasnt working with the variables but it may have been some cache on my computer